ghost/webmentions/test/MentionSendingService.test.js

```diff
@@ -426,41 +440,6 @@ describe('MentionSendingService', function () {
             assert(scope.isDone());
         });
 
-        // Redirects are currently not supported by got for POST requests!
-        //it('Can handle redirect responses', async function () {
-        //    const scope = nock('https://example.org')
-        //        .persist()
-        //        .post('/webmentions-test')
-        //        .reply(302, '', {
-        //            headers: {
-        //                Location: 'https://example.org/webmentions-test-2'
-        //            }
-        //        });
-        //    const scope2 = nock('https://example.org')
-        //        .persist()
-        //        .post('/webmentions-test-2')
-        //        .reply(201);
-        //
-        //    const service = new MentionSendingService({externalRequest});
-        //    await service.send({
-        //        source: new URL('https://example.com'),
-        //        target: new URL('https://example.com'),
-        //        endpoint: new URL('https://example.org/webmentions-test')
-        //    });
-        //    assert(scope.isDone());
-        //    assert(scope2.isDone());
-        //});
-        // TODO: also check if we don't follow private IPs after redirects
```

Comment:
I think we can uncomment the test, because it should pass

Reply:
Pulling this comment down here.

I saw some discussion within got docs/issues that post requests should not be redirected. Not sure if you have any background on that, but that added to the confusion for me here.

https://github.com/sindresorhus/got/issues/216#issuecomment-237610929

This is what has me concerned. I didn't get that test to work, and I created a small local npm repo with just `got` to try testing it and wasn't able to get anywhere with successfully following a redirect. I get an `ERR_NOCK_NO_MATCH` error. I haven't tried digging through the `got` codebase yet to track down why. Hoping you might have some insight into this before going down that path.

Reply:
See other comment